src/components/TrackSettings.js

```diff
@@ -12,7 +12,8 @@ import ColorPicker from "./ColorPicker";
  * trackColorSettings expects an object in the form of
  * {mainPalette: string,
  *  auxPallate: string,
- *  colorReadsByMappingQuality: boolean}
+ *  colorReadsByMappingQuality: boolean,
+ *  alphaReadsByMappingQuality: boolean}
```

Comment:
Maybe this should cross-reference the schema file https://github.com/vgteam/sequenceTubeMap/blob/0452ecb82d057372e359a9b456d789336e5ab8a1/src/Types.ts?

Reply:
Also, there's now two flags, which is a suspicious number of flags. We should think about redesigning this soon.

Really, the new alpha flag is the only one that ought to be a real flag, since it can be on or off and the main and aux palettes still matter.

If you turn on the color by mapping quality flag for a read track, the main and aux palettes are ignored. But keeping them around makes it easier to turn on color-by-MAPQ and then turn it off again and get back the same colors you had set.